jobs/aws/eks-distro-build-tooling/eks-distro-base-periodics.yaml

```diff
@@ -17,7 +17,7 @@ periodics:
   labels:
     image-build: "true"
   # Runs every weekday (M-F) at 11am PST
-  cron: "0 19 * * 1-5"
+  cron: "30 19 * * 1-5"
```

Comment:
In general, we probably want to pick slightly off times for jobs anyway so we don't have everything running on the hour